src/main/java/org/isf/menu/manager/UserSettingManager.java

```diff
@@ -35,6 +36,11 @@ public class UserSettingManager {
 	@Autowired
 	private UserSettingOperationRepository userSettingIoOperationRepository;
 
+	public UserSettingManager(UserSettingOperationRepository userSettingIoOperationRepository) {
+		super();
+		this.userSettingIoOperationRepository = userSettingIoOperationRepository;
```

Comment:
I am confused by this statement; the value of `userSettingIoOperationRepository` is already set because of `@Autowired'.   Is there some other use case that has a different repository object?

Reply:
no there is because i have firstly use dependance injection by constructor after the use @Autowired and forget to remove the constructor. I will remove it